### PR TITLE
Added alarms to telemetry fields and abstracts

### PR DESCRIPTION
**Pull request by EugeniaDickson**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -1562,23 +1839,29 @@ CHWPVM:
...
+  - chilled_water_flowrate_sensor
+  - chilled_water_valve_percentage_sensor
+  - cooling_thermal_power_capacity
   - discharge_air_temperature_sensor_1
   - discharge_air_temperature_sensor_2
   - leaving_cooling_coil_temperature_sensor
-  - cooling_thermal_power_capacity
-  - chilled_water_flowrate_sensor
-  - chilled_water_valve_percentage_sensor
+  - failed_discharge_air_temperature_alarm
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1855]: needs to be enumerated

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -1562,23 +1839,29 @@ CHWPVM:
...
+  - failed_zone_air_temperature_alarm
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1856]: enumeration

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -1725,60 +2060,81 @@ DX4ZC:
...
+  - cooling_thermal_power_capacity
   - discharge_air_temperature_sensor
   - leaving_cooling_coil_temperature_sensor
-  - cooling_thermal_power_capacity
-  - cooling_percentage_command
-  - compressor_speed_percentage_command
+  - compressor_lost_power_alarm
+  - failed_compressor_alarm
+  - failed_discharge_air_temperature_alarm
+  - failed_zone_air_temperature_alarm
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:2083]: enumeration

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 4]
```diff
@@ -73,6 +73,7 @@ descriptor:
   broken: "A state of disrepair; fractured or segmented."
   building: "Applies to the entire building or group of zones within building (e.g. Building_Air_Static_Pressure_Sensor)."
   bypass: "Route which fluid takes to bypass process."
+  cafe: "Applies for cafe areas within building"
```
Comment (cstirdivant) [inline ontology/yaml/resources/subfields/subfields.yaml:76]: Need a better description here

File: ontology/yaml/resources/subfields/subfields.yaml [hunk 5]
```diff
@@ -139,6 +140,7 @@ descriptor:
   input: "The input to a system."
   inter: "Represents something in between references (e.g. average_inter_line_voltage, the average voltage between lines)."
   isolation: "Process of isolating one component or process from another."
+  kitchen: "Applies for kitchen areas within building"
```
Comment (cstirdivant) [inline ontology/yaml/resources/subfields/subfields.yaml:143]: Please make a better description.

**Review by cstirdivant — CHANGES_REQUESTED** [review]
please check single quotes around the id strings - I believe they need to be double quotes

File: ontology/yaml/resources/subfields/subfields.yaml (hunk 4 above)
Comment (tasodorff) [inline ontology/yaml/resources/subfields/subfields.yaml:76]: iirc we decided to remove cafe and kitchen as descriptors.

**Review by cstirdivant — APPROVED** [review]

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**